### DISCUSSION TITLE
Add official support for php 8.3

### DISCUSSION THREAD

**Discussion by zip-fa** [nginx/unit#1069] [General]
Hi. Since there is no open issues about this topic, i’ll open this one.
PHP 8.3 image already released some time ago.
Adding its support to official unit docker image list will be much appreciated!

**Comment by tippexs** [comment]
Hi @zip-fa thanks for reaching out.

Currently - we do not have a process of releasing newer Docker Images for newer versions of a language as they are beeing released. This will be changed in the future BUT needs some extra time and actions. I would like to transform this issue into a discussion as this will be a better place to discus things and create concrete issues / work items.

In the mean time you can follow our process to build your own Docker-Image using our packaging tools.
https://unit.nginx.org/installation/#inst-lang-docker. Please see the information under _Customizing Language Versions in Docker Images_

Let me know if that helps.
